### PR TITLE
Continuously push prerelease builds to Azure Storage 

### PR DESCRIPTION
**Pull request by david-driscoll**

### CODE REVIEW

**Comment by david-driscoll** [comment]
So many commits, we should merge with a squash just to keep the history a little more tidy.

File: .travis.yml [hunk 1]
```diff
@@ -3,85 +3,92 @@ sudo: required
 dist: trusty
 env:
   global:
-    secure: m2PtYwYOhaK0uFMZ19ZxApZwWZeAIq1dS//jx/5I3txpIWD+TfycQMAWYxycFJ/GJkeVF29P4Zz1uyS2XKKjPJpp2Pds98FNQyDv3OftpLAVa0drsjfhurVlBmSdrV7GH6ncKfvhd+h7KVK5vbZc+NeR4dH7eNvN/jraS//AMJg=
+  - secure: m2PtYwYOhaK0uFMZ19ZxApZwWZeAIq1dS//jx/5I3txpIWD+TfycQMAWYxycFJ/GJkeVF29P4Zz1uyS2XKKjPJpp2Pds98FNQyDv3OftpLAVa0drsjfhurVlBmSdrV7GH6ncKfvhd+h7KVK5vbZc+NeR4dH7eNvN/jraS//AMJg=
```
Comment (rchande) [inline .travis.yml]: Should we doc what variables these are?

File: .travis.yml [hunk 2]
```diff
@@ -3,85 +3,92 @@ sudo: required
...
-    if [ "$TRAVIS_OS_NAME" == "linux" ]; then
-      rm artifacts/package/omnisharp-osx.tar.gz
-    elif [ "$TRAVIS_OS_NAME" == "osx" ]; then
-      rm artifacts/package/mono.tar.gz
-      rm artifacts/package/omnisharp-linux-x86.tar.gz
-      rm artifacts/package/omnisharp-linux-x64.tar.gz
-    fi
-
+- |
+  # Only deploy OSX-specific OmniSharp from the OSX.
```
Comment (rchande) [inline .travis.yml]: Does the mac build produce different binaries? If so, should we disable cross-compile?

File: README.md [hunk 3]
```diff
@@ -18,6 +18,23 @@ See our [change log](https://github.com/OmniSharp/omnisharp-roslyn/blob/master/C
...
+* Version is auto incremented and is visible in the travis or appveyor build output
+* Package Name would be either `omnisharp` or `omnisharp.http`
+* `os/arch` will be one of the following:
+  * `win-x64`
+  * `win-x86`
+  * `linux-x64`
+  * `linux-x86`
+  * `osx`
+  * `mono`  (Requires global mono installed)
+* Extenisons is archive specific, windows will be `zip` and other willth `tar.gz`.
```
Comment (rchande) [inline README.md]: Extensions are ... others will be

File: build.cake [hunk 4]
```diff
@@ -106,7 +105,7 @@ Task("Setup")
     .IsDependentOn("ValidateMono")
     .IsDependentOn("InstallDotNetCoreSdk")
     .IsDependentOn("InstallMonoAssets")
-    .IsDependentOn("CreateMSBuildFolder");
+    .IsDependentOn("CreateMSBuildFolder");;
```
Comment (rchande) [inline build.cake]: Extra ;

File: README.md (hunk 3 above)
Comment (david-driscoll) [reply README.md]: whats english? 😢

**Pull request merged**